### ISSUE TITLE
texture creation fails after import during material redefinition

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#339]
**To reproduce:** create two files, `import.yaml` and `style.yaml`.

In `import.yaml` specify a style like so:

```yaml
styles:
    test-style:
        base: polygons
        material:
            diffuse: .75
```

in `style.yaml`, override the diffuse parameter with a texture, like so:

```yaml
styles:
    test-style:
        material:
            diffuse:
                texture: sofa.png
```

**Result:**
The scene will fail to load, with:

> Scene.load() failed to load http://localhost:8080/unit-tests/materials/materials-texture.yaml: Cannot create property 'texture' on number '0.75' TypeError: Cannot create property 'texture' on number '0.75'(…)

**Expected result:**
The material definition specified in `style.yaml` will override the initial `diffuse` setting and use the texture rather than the `.75` setting.

**Comment by bcamper** [comment]
Makes sense, will fix.

**Comment by bcamper** [comment]
@meetar do you have a test scene handy I can use for this?

**Comment by bcamper** [comment]
Should be fixed by https://github.com/tangrams/tangram/commit/c540943eb2f0c63f268e22e38afd61ba51177ad4, but please let me know when you can verify.

**Comment by bcamper** [comment]
released in v0.8.1

**Comment by meetar** [comment]
:+1: Confirmed: http://tangrams.github.io/differ/?1=http%3A//tangrams.github.io/differ-tests/unit-tests/materials/materials01.json&2=http%3A//tangrams.github.io/differ-tests/unit-tests/materials/materials01.json&lib1=0.7&lib2=0.8&go
